src/allmydata/web/storage.py

```diff
@@ -16,91 +22,108 @@ def remove_prefix(s, prefix):
     return s[len(prefix):]
 
 
-class StorageStatus(MultiFormatPage):
-    docFactory = getxmlfile("storage_status.xhtml")
-    # the default 'data' argument is the StorageServer instance
+class StorageStatusElement(Element):
+    """Class to render a storage status page."""
+
+    loader = XMLFile(FilePath(__file__).sibling("storage_status.xhtml"))
 
     def __init__(self, storage, nickname=""):
-        rend.Page.__init__(self, storage)
+        """
+        :param _StorageServer storage: data about storage.
+        :param string nickname: friendly name for storage.
+        """
+        super(StorageStatusElement, self).__init__()
         self.storage = storage
-        self.nickname = nickname
+        self.nick = nickname
 
```

Comment:
If we're changing this name anyway, maybe this should be `self._nick` to indicate it's "private" (if that is indeed the case).

Reply:
The `nickname` class variable had a name conflict with the method `nickname()`, which is a `@renderer`.  (It confused template expansion, I think.)  I will update this name.

Reply:
Renamed them in eec7dbb.  I'm not sure ci/circleci: another-locale fails now.

Reply:
The failure is:

> twisted.internet.error.CannotListenError: Couldn't listen on any:38325: [Errno 98] Address already in use.

which is a common flavor of spurious intermittent failure in the test suite. :/    You can ignore it for the purposes of this PR.